cashkick-frontend/src/App.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { ThemeProvider } from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
+import theme from "../themes/index";
+import Button from "./components/atoms/Button/index";
+
+const App = () => {
+  const handleClick = () => {
+    alert("Button clicked!");
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <CssBaseline />{" "}
+      <Button
+        buttonText="Click Me"
+        onClickEvent={handleClick}
+        propVariant="contained"
+        customColor="#4CAF50"
```

Comment:
please make sure that you have all the hex codes defined at one place(in the theme file) and you import them.